the-journal-of-veterinary-medical-science.csl

```diff
@@ -13,8 +13,8 @@
     </author>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <issn>0021-4922</issn>
-    <eissn>1347-4065</eissn>
+    <issn>0916-7250</issn>
+    <eissn>1347-7439</eissn>
```

Comment:
@shojitakahashi, the original ISSNs appear to be from the "Japanese Journal of Applied Physics" (compare http://iopscience.iop.org/journal/1347-4065 and https://www.jstage.jst.go.jp/browse/jvms).